XIVComboExpanded/Combos/PCT.cs

```diff
@@ -286,6 +286,11 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     return PCT.RainbowDrip;
                 }
 
+                if (IsEnabled(CustomComboPreset.PictomancerHolyHammerCombo) && HasEffect(PCT.Buffs.HammerReady))
+                {
+                    return PCT.HammerStamp;
```

Comment:
@MKhayle before you release this, this line will need to be an `OriginalHook()` call to properly use the entire hammer chain, I believe.